CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+
+Pull Request Philosophy
+-----------------------
+
+Patchsets should always be focused. For example, a pull request could add a feature, fix a bug, or refactor code; but not a mixture. Please also avoid super pull requests which attempt to do too much, are overly large, or overly complex as this makes review difficult.
+
+
+###Features
+
+When adding a new feature, thought must be given to the long term technical debt and maintenance that feature may require after inclusion. Before proposing a new feature that will require maintenance, please consider if you are willing to maintain it (including bug fixing). If features get orphaned with no maintainer in the future, they may be removed by the Repository Maintainer.
+
+
+###Refactoring 
+
+Refactoring is a necessary part of any software project's evolution. The following guidelines cover refactoring pull requests for the project.
+
+There are three categories of refactoring, code only moves, code style fixes, code refactoring. In general refactoring pull requests should not mix these three kinds of activity in order to make refactoring pull requests easy to review and uncontroversial. In all cases, refactoring PRs must not change the behaviour of code within the pull request (bugs must be preserved as is).
+
+Project maintainers aim for a quick turnaround on refactoring pull requests, so where possible keep them short, uncomplex and easy to verify. 
+
+
+"Decision Making" Process
+-------------------------
+
+The following applies to code changes to the Bitcoin Core project (and related projects such as secp256k1), and is not to be confused with overall Bitcoin Network Protocol consensus changes.
+
+Whether a pull request is merged into Bitcoin Core rests with the project merge maintainers and ultimately the project lead. 
+
+Maintainers will take into consideration if a patch is in line with the general principles of the project; meets the minimum standards for inclusion; and will judge the general consensus of contributors.
+
+In general, all pull requests must:
+
+  - have a clear use case, fix a demonstrable bug or serve the greater good of the project (for example refactoring for modularisation);
+  - be well peer reviewed;
+  - have unit tests and functional tests where appropriate;
+  - follow code style guidelines;
+  - not break the existing test suite;
+  - where bugs are fixed, where possible, there should be unit tests demonstrating the bug and also proving the fix. This helps prevent regression;
+  - where consensus rules are affected, must be accompanied by a published BIP assigned with a number;
```

Comment:
absolutely. They are a wholly different animal from normal run-of-the-mill changes, that should be made clear.